src/rpc/mining.cpp

```diff
@@ -731,7 +731,10 @@ UniValue getblocktemplate(const UniValue& params, bool fHelp)
 
         if (certSupported) {
             CCoinsViewCache view(pcoinsTip);
-            pblock->hashScTxsCommitment = pblock->BuildScTxsCommitment(view);
+            pblock->hashScTxsCommitment.SetNull();
```

Comment:
Is it ok no call to `BuildScTxsCommitmentGuard` is performed here, compared for example to analogous code in `getblockmerkleroots`?

Reply:
Yes, it is safe, as we are building a commitment out a set of tx/cert that we know are safe, as they have been previously chosen by `CreateNewBlockWithKey()`, where a similar - actually more in-depth - check has been performed.